ppl/zqd/db/postgresdb/postgresdb.go

```diff
@@ -8,18 +8,25 @@ import (
 	"github.com/brimsec/zq/ppl/zqd/db/schema"
 	"github.com/brimsec/zq/zqe"
 	"github.com/go-pg/pg/v10"
+	"go.uber.org/zap"
 )
 
 type PostgresDB struct {
-	db *pg.DB
+	db     *pg.DB
+	logger *zap.Logger
 }
 
-func Open(ctx context.Context, conf Config) (*PostgresDB, error) {
+func Open(ctx context.Context, logger *zap.Logger, conf Config) (*PostgresDB, error) {
+	logger = logger.With(
```

Comment:
I don't think these need to be printed with each database log line; how about just adding them to the 'Connected' log below?